latest-langs

```diff
@@ -14,6 +14,7 @@ constant %paths = (
     'COBOL'      => 'en.wikipedia.org/wiki/GnuCOBOL',
     'Coconut'    => 'github.com/evhub/coconut/releases/latest',
     'Clojure'    => 'github.com/babashka/babashka/releases/latest',
+    'Civet'      => 'npmjs.com/package/@danielx/civet/v/latest',
```

Comment:
Did you run this file? You'll need to add npmjs.com to the switch in the code if you want to support that domain.

Reply:
[Update latest-langs to support npmjs.com](https://github.com/code-golf/code-golf/pull/1249/commits/1b3bad52929343ed554462a6da1c8d95b2fd9002)

![image](https://github.com/user-attachments/assets/a0adc29f-ad5c-4904-9101-321de038921f)